docs/content/en/docs/getting-started.md

```diff
@@ -29,6 +29,13 @@ Start integrating GoodData into your Python application right now.
     sdk = GoodDataSdk.create(host, token)
     ```
 
+    Alternatively, you can create the instance from a config file:
+
+    ```python
+    # The SDK can be initialized from a config file, so you don't have to hardcore the host and token.
+    sdk = GoodDataSdk.create_from_profile("Profile1", profile_path)
+    ```
+
```

Comment:
It would be handy to place a content of config file with several profile and note that `default` profile is used by default.

Reply:
Let's open it up during our meeting.

Reply:
+ rename profile_path -> profiles_path